qutip/qip/circuit.py

```diff
@@ -45,46 +47,50 @@ class Gate(object):
     """
     Representation of a quantum gate, with its required parametrs, and target
     and control qubits.
+    
+    Parameters
+    ----------
+    name : String
```

Comment:
lower capital unless this is a class. String -> string. Same applies for other parameters belows.